modules/opensearch/opensearch.client.inc

```diff
@@ -732,11 +425,130 @@ function opensearch_execute($request) {
     }
     watchdog('ting client', 'Error performing request: ' . $e->getMessage(), NULL, WATCHDOG_ERROR, 'http://' . $_SERVER["HTTP_HOST"] . request_uri());
     return FALSE;
-  } finally {
+  }
+  finally {
     timer_stop('ting');
   }
 }
 
+/**
+ * Try and predict future search requests and warm up cache.
+ *
+ * Base on the collections given and the entity model for ting objects we expect
+ * the next search requests to be getCollectionRequest and getObjectRequest
+ * without relations. The information to populate the cache with response to
```

Comment:
...  but `opensearch_execute` will always enable relations given the change on L352?

Reply:
Updated the comment